<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWebTestClass</class>
 <widget class="QWidget" name="QtWebTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWebTest</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>后退</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_front">
       <property name="text">
        <string>前进</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_refresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_url"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_go">
       <property name="text">
        <string>GO</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWebTest.qrc"/>
 </resources>
 <connections/>
</ui>
